webapp/.eslintrc.json

```diff
@@ -70,7 +70,7 @@
         "no-var": ["error"],
         "no-with": ["error"],
         "prefer-const": ["error"],
-        "prefer-promise-reject-errors": ["error"],
+        "prefer-promise-reject-errors": "off",
```

Comment:
This will be handled once we use the common ESlint rules. It is ok for now

Reply:
ok, then you'll need to align it to a right practice.